webpages/settings/index.js

```diff
@@ -624,7 +602,7 @@ let fuse;
     }
   });
 
-  document.title = chrome.i18n.getMessage("settingsTitle");
+  document.title = getMessage("settingsTitle");
```

Comment:
Again, the responsibility of setting the tab title should be separate and would not belong here.